init.sh

```diff
@@ -0,0 +1,22 @@
+# This script is used to initialise the dev env for this package.
```

Comment:
Why do we need this file? This was in auth-react cause there it was failing with node 16 if we only did `npm i -d`. Does this fail here too?

Reply:
We dont, removed the script file and the package.json script